packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -276,21 +229,29 @@ const props = defineProps<{
 const emit = defineEmits(['update-project']);
 
 const tabStore = useTabStore();
-const router = useRouter();
-const resources = useResourcesStore();
 
-const newModelId = ref<string>('');
-const isNewModel = ref<boolean>(true);
+// Makes asset from workflow node appear
+const openedWorkflowNodeStore = useOpenedWorkflowNodeStore();
+const workflowNodeAssetId = computed(() => {
```

Comment:
Can you add some comments to explain what is happening here? I find it a bit hard to follow.